classes/options-general.php

```diff
@@ -526,13 +526,13 @@ class='howto'><?php _e( 'Select "No Limit" to display all results within the sea
 																echo '<br /><small style="color:red";><em>' . $disabledmsg . '</small></em>';
 															} ?>
 
-															<!--<br /><label for="lock_default_location" id="lock_default_location_label"><input type="checkbox" name="lock_default_location" id="lock_default_location" value="1" <?php checked( $lock_default_location ); ?> /> <?php _e( 'Stick to default location set above', 'simplemap' ); ?></label>-->
+															<!--<br /><label for="lock_default_location" id="lock_default_location_label"><input type="checkbox" name="lock_default_location" id="lock_default_location" value="1" <?php checked( $lock_default_location ); ?> /> <?php _e( 'Stick to default location set above', 'SimpleMap' ); ?></label>-->
 														</td>
 													</tr>
 
 													<tr valign="top">
 														<td><label
-																for="zoom_level"><?php _e( 'Default Zoom Level', 'simplemap' ); ?></label>
+																for="zoom_level"><?php _e( 'Default Zoom Level', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 138 characters

<!-- Comment by [SideCI](https://sideci.com) -->